manifests/p/Parsec/Parsec/150-84c/Parsec.Parsec.installer.yaml

```diff
@@ -2,7 +2,7 @@
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
 
 PackageIdentifier: Parsec.Parsec
-PackageVersion: "150-82a"
+PackageVersion: "150-84c"
```

Comment:
```suggestion
PackageVersion: "150-84b"
```
This must match Control Panel.

Reply:
![image](https://user-images.githubusercontent.com/11600822/182004087-d458db56-354b-4612-87e6-da07de41f6d8.png)